.changeset/polite-donuts-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend': patch
+---
+
+Internal refactors as a path toward #18062
```

Comment:
Probably better to explain here what #18062 is with a few words :p

Reply:
aight, updated